ic10emu/src/interpreter.rs

```diff
@@ -2626,80 +214,87 @@ pub fn i64_to_f64(i: i64) -> f64 {
 
 #[cfg(test)]
 mod tests {
-    use crate::vm::VMError;
-
-    use super::*;
 
```

Comment:
don't forget these tests!

Reply:
opp! your right. I commented them out during the rebuild because the interface changed a lot.
Fixed and passing.